src/org/usfirst/frc/team2175/command/group/DriveInchesWithoutEncodersCommandGroup.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc.team2175.command.group;
+
+import org.usfirst.frc.team2175.ServiceLocator;
+import org.usfirst.frc.team2175.command.single.ArcadeDriveForAutonCommand;
+import org.usfirst.frc.team2175.subsystem.drivetrain.DrivetrainSubsystem;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class DriveInchesWithoutEncodersCommandGroup extends CommandGroup {
+    private DrivetrainSubsystem drivetrainSubsystem;
+    private final double inches;
+    private final double percentVbus;
+    private final double seconds;
+
+    public DriveInchesWithoutEncodersCommandGroup(final double inches,
+            final double percentVbus) {
+        this.inches = inches;
+        drivetrainSubsystem = ServiceLocator.get(DrivetrainSubsystem.class);
+        this.percentVbus = percentVbus;
+        seconds = calculateSecondsFromInches(inches, percentVbus);
+        addSequential(new ArcadeDriveForAutonCommand(percentVbus, 0), seconds);
+    }
+
+    private double calculateSecondsFromInches(double inches,
+            double percentVbus) {
+        double seconds;
+        final double INCHES_PER_SECOND_ZERO_POINT_SIX = 22.6008;
+        final double INCHES_PER_SECOND_ZERO_POINT_SEVEN = 24.375;
+        final double INCHES_PER_SECOND_ZERO_POINT_EIGHT = 42.6;
```

Comment:
@NoahConnors When we have more time to test (i.e. after Duluth), I'd like to come up with better numbers / equations to use here.